src/Core/Cache/MemcachedCacheDriver.php

```diff
@@ -10,8 +10,11 @@
  *
  * @author Hypolite Petovan <mrpetovan@gmail.com>
  */
-class MemcachedCacheDriver extends BaseObject implements ICacheDriver
+class MemcachedCacheDriver extends BaseObject implements IMemoryCacheDriver
 {
+	use TraitCompareSet;
```

Comment:
Is there a reason why you only use these traits here?

Reply:
Yes, redis has a native implementation of "compare and set" algorithm. Whereas for memcache(d) , we have to rebuild such a feature (=> the traits).

The compare and set algorithm is necessary for distributed systems to avoid race conditions.

Reply:
Do you expect other lock drivers to make use of these traits?